src/Tizen.NUI/src/public/Capture.cs

```diff
@@ -0,0 +1,294 @@
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+namespace Tizen.NUI
+{
+    using global::System;
+    using global::System.ComponentModel;
+    using global::System.Runtime.InteropServices;
+    using global::System.ComponentModel;
+    using Tizen.NUI.BaseComponents;
+
+    /// <summary>
+    /// Capture snapshots the current scene and save as a file.
+    /// Applications should follow the example below to create capture :
+    /// <code>
+    /// Capture capture = new Capture();
+    /// </code>
+    /// If required, you can also subscribe Finished event :
+    /// <code>
+    /// capture.Finished += onCaptureFinished;
+    /// </code>
+    /// At the subcribed event handler, user can know whether capture finish succeeded state.
+    /// <code>
+    /// private void onCaptureFinished(object sender, CaptureFinishStateEventArgs e)
+    /// {
+    ///   if(e.Success) { //capture success, do something. }
+    ///   else { //capture failure, do something. }
+    /// }
+    /// </code>
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class Capture : BaseHandle
+    {
+        /// <summary>
+        /// Create an Capture.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public Capture() : this(Interop.Capture.New(), true)
+        {
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+        }
+
+        internal Capture(IntPtr cPtr, bool cMemoryOwn) : base(Interop.Capture.Upcast(cPtr), cMemoryOwn)
+        {
+        }
+
+        /// <summary>
+        /// Dispose
+        /// </summary>
+        /// <param name="type"></param>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        protected override void Dispose(DisposeTypes type)
+        {
+            if (disposed)
+            {
+                return;
+            }
+
+            if (type == DisposeTypes.Explicit)
+            {
+                //Called by User
+                //Release your own managed resources here.
+                //You should release all of your own disposable objects here.
+            }
+
+            base.Dispose(type);
+        }
+
+        /// This will not be public opened.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        protected override void ReleaseSwigCPtr(HandleRef swigCPtr)
+        {
+            Interop.Capture.Delete(swigCPtr);
+        }
+
+        /// <summary>
+        /// Generate copy constructor.
+        /// </summary>
+        /// <param name="source">A reference to the copied handle</param>
+        /// <returns></returns>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public static Capture Assign(Capture source)
```

Comment:
I can't find "assign" function in native Capture api. I think "assign" is not common name for public api.

Reply:
yes, I was wrong. it was not copy constructor. it was '=' operator override in dali. this is not used in nui so I removed this method. thank you.